src/monodroid/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ else()
   set(ANALYZERS_ENABLED OFF)
 endif()
 
-option(ENABLE_NET "Enable compilation for .NET 6+" OFF)
+option(ENABLE_NET "Enable compilation for .NET 6+" ON)
```

Comment:
`ENABLE_NET` use could be removed completely. We no longer need to build for the desktop, so with classic gone `.NET` is the only target

Reply:
Done, makes sense to clean this up now.